content/200-orm/100-prisma-schema/40-location.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: Schema Location
+metaTitle: Prisma Schema Location and Configuration
+metaDescription: Documentation regarding proper location of Prisma Schema including default naming and multi-file schema.
+---
+
+## Default Prisma Schema location
+
+The Prisma CLI looks for the Prisma schema in the following locations, in the following order:
+
+1.  The location specified by the [`--schema` flag](/orm/reference/prisma-cli-reference), which is available when you `introspect`, `generate`, `migrate`, and `studio`:
+
+    ```terminal
+    prisma generate --schema=./alternative/schema.prisma
+    ```
+
+2.  The location specified in the `package.json` file (version 2.7.0 and later):
+
+    ```json
+    "prisma": {
+      "schema": "db/schema.prisma"
+    }
+    ```
+
+3.  Default locations:
+
+    - `./prisma/schema.prisma`
+    - `./schema.prisma`
+
+The Prisma CLI outputs the path of the schema that will be used. The following example shows the terminal output for `prisma db pull`:
+
+```no-lines
+Environment variables loaded from .env
+//highlight-next-line
+Prisma Schema loaded from prisma/schema.prisma
+
+Introspecting based on datasource defined in prisma/schema.prisma …
+
+✔ Introspected 4 models and wrote them into prisma/schema.prisma in 239ms
+
+Run prisma generate to generate Prisma Client.
+```
+
+## Prisma Schema naming
+
+The default name for the schema is `schema.prisma`. When your schema is named like this, the Prisma CLI will detect it automatically in the directory where you invoke the CLI command (or any of its subdirectories).
```

Comment:
```suggestion
The default name for the schema is `schema.prisma`.
```
The half sentence that I removed is super weird, and the "subdirectories" bit even wrong, right? Maybe just remove it?